plugin/src/openshift/components/KialiController.tsx

```diff
@@ -90,7 +90,7 @@ class KialiControllerComponent extends React.Component<KialiControllerProps> {
 
   render() {
     return this.state.configLoaded ? (
-      <div className={`${globalStyle} ${ossmcStyle} ${varStyle.kiali}`}>{this.props.children}</div>
+      <div className={`${globalStyle} ${ossmcStyle} ${cssVariables.style}`}>{this.props.children}</div>
```

Comment:
Is this commit "change cssVariables name" something you have to do every time? If so, should we add this to the copy script so it isn't manual?

If this is just a one-off change, then never mind :)